decidim-conferences/spec/system/conference_registrations_spec.rb

```diff
@@ -128,17 +128,34 @@ def visit_conference_registration_type
       end
     end
 
-    context "and there is registrations types" do
+    context "and there is published registrations types" do
       it "allows to register" do
         visit_conference
         within ".conference__hero" do
           expect(page).to have_content "Register"
+        end
+        within ".conference__content-block" do
+          expect(page).to have_content "Register"
           click_on "Register"
         end
         expect(page).to have_content "CHOOSE YOUR REGISTRATION OPTION:"
       end
     end
 
+    context "and there is unpublished registrations types" do
+      let(:registration_types) { :unpublished }
```

Comment:
```suggestion
                let!(:registration_types) do
            create_list(:registration_type, 5, :unpublished, conference:)
          end
```

registration_types must be an array of unpublished registration_types rather than a symbol ":unpublished"